dronegen/common.go

```diff
@@ -17,6 +17,11 @@ var (
 		Ref:   triggerRef{Include: []string{"refs/tags/v*"}},
 		Repo:  triggerRef{Include: []string{"gravitational/*"}},
 	}
+	triggerBuildbox = trigger{
```

Comment:
nit: this is very similar to `triggerPush`
how about renaming it to `triggerPushMasterOnly`